src/tracer.js

```diff
@@ -80,6 +80,9 @@ export default class Tracer {
             } else {
                 fields.operationName = nameOrFields;
             }
+            if (fields.parent) {
+                fields.parent = fields.parent._imp;
```

Comment:
I think this can be done in the implementation. I am not a fan of a function modifying the dictionary it receives as a parameter. For that matter, not happy with L81, but it's a different discussion (on gitter).